apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveItem.tsx

```diff
@@ -59,7 +60,7 @@ const SuspendableMoveItem = ({
       pl="2.25rem"
       size="xs"
       onClick={handleOnClick}
-      leftIcon={<BiFolder />}
+      leftIcon={type === "Collection" ? <BiData /> : <BiFolder />}
```

Comment:
nit: I think I've seen this if-else a couple of times, might be good to standardise to use a common function that determines the icon to use

For example, `ICON_MAPPINGS` inside `apps/studio/src/features/dashboard/components/DirectorySidebar/constants.ts` has this mapping for all the resources currently.

Reply:
udpated